### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by srush** [ibm-aur-nlp/PubLayNet#24]
Thanks so much for this repo it has been amazingly useful. We used it to build ICLR 2020 virtual addition and added all the pictures this way!

https://twitter.com/srush_nlp/status/1253788694739386371

**Comment by zhxgj** [comment]
Hi @srush, Thanks for using PubLayNet. Glad to hear it helped!

**Comment by zhxgj** [comment]
Hi @srush We happen to be preparing a blog about PubLayNet and want to add this great news to the blog. Do you have an estimate of how much time you think PubLayNet saved you? Any type of metric will be greatly useful for us. Thanks very much.

**Comment by srush** [comment]
Infinity time. I tried every other direct PDF extraction method and they all had intractable issues, e.g. couldn't extract PDF images or were too low res or were just bad. I was about to give up until I found this tool, and in 20 lines of code, and 2 hours on Google Colab (sorry) , I had every image from 700 papers at high res with precise enough accuracy (it's unfortunately bad at columns? Guessing that is because of pubmed).

**Comment by zhxgj** [comment]
Thanks @srush for your feedback. Do you have any examples where the model is bad at columns? I can have a look if that is because of any bias or annotation errors in the data, if I can fix it.

**Comment by srush** [comment]
I don't have an exact example, but it seemed to do worse in papers like
this where the text wraps around the images

https://openreview.net/pdf?id=Bkxv90EKPB

**Comment by zhxgj** [comment]
Ah, yes. The dataset does not have many samples with text wraps around images. Most journals do not typeset in that way. And I also think our automated annotation algorithm does not handle this case well, and poor annotations are excluded, which further reduces samples with this appearance.

**Comment by zhxgj** [comment]
> Infinity time, I would have given up. I tried every other direct PDF extraction method and they all had intractable issues, e.g. couldn't extract PDF images or were too low res or were just bad. I was about to give up until I found this tool, and in 20 lines of code, and 2 hours on Google Colab (sorry) , I had every image from 700 papers at high res with precise enough accuracy (it's unfortunately bad at columns? Guessing that is because of pubmed).

Hi @srush , could I please quote your above feedback in our blog post?

**Comment by srush** [comment]
Yes you can use whatever you need.

Right, I figured this was non standard formatting. For whatever reason you
do see it sometimes in NeurIPS, ICLR. For future years maybe we should
generate a small finetuning set.

**Comment by zhxgj** [comment]
Thanks @srush Yes, it is a good idea to have a small fine-tuning set for a specific template. The set can be pre-annotated with our model then manually curated, which will save some time.

**Comment by tshrjn** [comment]
Hi there, 

This is great! However, I just wanted to clarify why something like [pyMupdf](https://github.com/pymupdf/PyMuPDF) or [pdfminer](https://pdfminersix.readthedocs.io/en/latest/howto/images.html) couldn't be used?
[Here](https://towardsdatascience.com/extracting-images-from-a-pdf-using-python-45bbb3f41e51)'s an example blog with a script for the same.

**Comment by srush** [comment]
Publaynet extracts the images as they are shown in the paper (cropped, captioned etc) which is more interesting and harder.
